src/index.html

```diff
@@ -10,13 +10,110 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <div class="header-wrapper">
+          <a
+            href="#"
+            class="logo"
+          >
```

Comment:
You need to code style everywhere if you have only 2 attributes
```suggestion
          <a href="#" class="header__logo">
```

Reply:
I changed settings in settings.json, in .prettierrc, but in the end I disabled the file formatting. Is there anything I can do to avoid this so that I can continue to have formatting?